etc/build-binary.bash

```diff
@@ -30,6 +30,11 @@ etcdeliver=${etc}/deliver
 prefix=${var}/local
 lib=${prefix}/lib
 
+if [ ${var} ]
```

Comment:
I still think you mean 

```
if [[ -d ${var} ]]
```
as your test just tests whether `var` evaluates to a non-zero length string.

Reply:
Oh, of course. That peace is from Paul's `production.bash`, and I didn't take a closer look.